module/__mocks__/foundry.js

```diff
@@ -214,6 +228,14 @@ global.ui = {
   notifications: new Notifications()
 }
 
+/**
+ * Global helper functions function
+ */
+// TODO: This really needs to be the real Foundry mergeObject function or there will be side effects that make the tests invalid
+global.mergeObject = function (original, other = {}) {
```

Comment:
I'm not sure what to do about this. I don't want to break the tests, but I can't think of a good approach.

One option would be to import foundry.js and cherrypick this sort of function into the global namespace as needed. However, that would prevent the tests running in CI (no Foundry) and relies on the developer pointing the test harness so it's still awful.